gradle/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM gradle
 
 # setup okteto message
-COPY bashrc /home/gradle/.bashrc
```

Comment:
Did you test this? When I did it the gradle image wasn't using root , no idea f this could cause a problem.

Reply:
It is root now